files/image_config/caclmgrd/caclmgrd

```diff
@@ -204,6 +204,19 @@ class ControlPlaneAclManager(object):
                                   ("DST_IP" in rule_props and rule_props["DST_IP"])):
                                 table_ip_version = 4
 
+                        if ((("SRC_IPV6" in rule_props and rule_props["SRC_IPV6"]) or
+                            ("DST_IPV6" in rule_props and rule_props["DST_IPV6"])) and
+                            (table_ip_version == 4)):
+                            log_error("CtrlPlane ACL table {} is a IPv4 based table and rule {} is a IPV6 rule!"
+                                                          .format(table_name, rule_id))
+                            sys.exit(1)
```

Comment:
Try to raise exception instead of exit(1). In this case the caller of this method will be able to catch the exception to perfrom additional operations before shutting down (e.g. some cleanup)

Reply:
addressed in the next commit.